### PR TITLE
Extract mask bytes function.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by chriseth** [comment]
It turns out this function is only useful in one place currently (will change with subsequent PRs). The second place I thought it would be useful masks starting from the right end of the value.

**Comment by chriseth** [comment]
Failure is SMT related.

File: libsolidity/codegen/YulUtilFunctions.h [hunk 1]
```diff
@@ -99,6 +99,11 @@ class YulUtilFunctions
 	/// signature: (value, shiftBytes, toInsert) -> result
 	std::string updateByteSliceFunctionDynamic(size_t _numBytes);
 
+	/// Function that sets all but the first ``bytes`` bytes of ``value`` to zero.
+	/// @note ``bytes`` has to be small enough not to overflo ``8 * bytes``.
```
Comment (hrkrshnn) [inline libsolidity/codegen/YulUtilFunctions.h]:
```suggestion
	/// @note ``bytes`` has to be small enough not to overflow ``8 * bytes``.
```

**Review by hrkrshnn — COMMENTED** [review]
Looks good except the typo.

**Pull request merged**